docs/src/components/ComponentPlayground/ComponentPlayground.tsx

```diff
@@ -27,6 +27,7 @@ const unsupportedComponents = [
   'Portal',
   'Provider',
   'RadioGroup',
+  'TextArea',
```

Comment:
There is a problem (along the lines: "A component is changing an uncontrolled input of type text to be controlled error in ReactJS") when one plays with `placeholder` in the playground. So I decided to disable it.

Reply:
Can be reenabled now?

Reply:
Since there is no longer `placeholder` prop on (nor `maxLength` prop), it can be enabled. But I am not sure about its value